benchmarks/apfel_bench.py

```diff
@@ -94,7 +94,8 @@ def benchmark_msbar(self, pto):
         when  passing kthr != 1 both apfel and eko use ``kThr * msbar``,
         as thr scale, where ``msbar`` is the usual ms_bar solution.
         However apfel and eko mange the alpha_s thr differently
-        (apfel uses the given mass parameters as thr), so the
+        (apfel uses the given mass parameters as thr multipied by the kthr),
+        while in eko only the already computed thr matters, so the
```

Comment:
*while* sounds weird, since the sentence is opposed to is in parentheses: I'd suggest to remove the parentheses above